src/styles/theme.js

```diff
@@ -32,9 +32,12 @@ const theme = createMuiTheme({
                 overflow: 'initial'
             },
         },
-        MuiTooltip: {
-            tooltip: {
-                fontSize: '1em'
+        MuiTypography: {
+            h4: {
+                fontSize: '1.75rem',
+                '@media (min-width:376px)': {
+                    fontSize: '2.125rem'
+                }
```

Comment:
Here we want to add `MuiTypography.h4` without removing `MuiTooltip.tooltip` right? or was it intentionally removed?